### ISSUE TITLE
Missing comma in CLI help text

### ISSUE THREAD

**Issue by kaiyoma** [lesshint/lesshint#387]
In version 3.3.1, the `linters` line is missing a comma after the `-l`, which throws off the formatting a bit:

```
...
    -V, --version              output the version number
    -c, --config [path]        configuration file path
    -e, --exclude [pattern]    path pattern to exclude
    -l --linters [linters]     paths to custom linters
    -r, --reporter [reporter]  reporter to use (<PATH>|stylish), defaults to "stylish"
    -x, --max-warnings [int]   Number of warnings to trigger nonzero exit code, defaults to 0
```

**Comment by jwilsson** [comment]
Thought I had fixed that! I'll get right on it!

**Comment by jwilsson** [comment]
And fixed in 49f560fee444a6aee152b56e26102a32bd5d9cbb.
